doc/doxygen/Doxylocal

```diff
@@ -1,3 +1,11 @@
+# This file is part of the dune-stuff project:
+#   https://github.com/wwu-numerik/dune-stuff
+# The copyright lies with the authors of this file (see below).
+# License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+# Authors:
+#   Rene Milk (2015)
+#
```

Comment:
This should be an empty line since what follows is a one-line comment and not a header comment